### PR TITLE
feat(replay): Performance table tab

### PR DESCRIPTION
**Pull request by ryan953**
v0 of the performance tab:
SCR-20230817-ogvt

Fixes https://github.com/getsentry/team-replay/issues/160

### CODE REVIEW

File: static/app/utils/number/toPixels.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Covnert a number into a `${number}px` value
```
Comment (billyvg) [inline static/app/utils/number/toPixels.tsx:2]:
```suggestion
 * Convert a number into a `${number}px` value
```

Comment (ryan953) [reply static/app/utils/number/toPixels.tsx]: oh, need to rebase this file away.

File: static/app/utils/replays/hooks/useActiveReplayTab.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ export enum TabKey {
 }
 
 function isReplayTab(tab: string, organization: Organization): tab is TabKey {
-  const hasPerfTab = organization.features.includes('session-replay-trace-table');
+  const hasPerfTab = true || organization.features.includes('session-replay-trace-table');
```
Comment (billyvg) [inline static/app/utils/replays/hooks/useActiveReplayTab.tsx:21]: Don't forget this

Comment (ryan953) [reply static/app/utils/replays/hooks/useActiveReplayTab.tsx:21]: fixed

**Review by michellewzhang — APPROVED** [review]
gif

**Pull request merged**